chevah/brink
# [#2640] Fix clean task for rhel4.

Problem
=======

Find from rhel 4 does not support +

Changes
========

Use simple print and xarg for RHEL4

How to test
=========

reviewers: @dumol 

run `paver test_remote rhel-4 --properties=force_clean=yes --wait` and see that  all is ok.

Thanks!
